### DISCUSSION TITLE
How to customize cloudflare worker script

### DISCUSSION THREAD

**Discussion by hlassiege** [nitrojs/nitro#623] [Q&A] [answered]
Hi,

I use Nuxt to build an app and I've setup nitro to use a "cloudflare" preset.
The build produces an index.mjs that will be used as worker script. 

Is it possible to configure something in order to modify the worker behaviour ?
For example I'd like to keep the control over the worker for AB testing purpose (read a cookie to route the traefik elsewhere in some case)

**Comment by Hebilicious** [comment, marked as answer]
Hi, you can have full control over the worker by using a custom preset. 
We're still working on the documentation, but you can find some information there https://github.com/unjs/nitro/issues/1373
